indexstyle.css

```diff
@@ -99,58 +106,85 @@ li a:hover {
 
 .container {
 	display: flex;
-	height: 100vh;
-	justify-content: space-between;
+	height: 90vh;
+	justify-content: space-evenly;
 	flex-wrap: wrap;
 	align-items: center;
-	padding: 20px;
+	padding: 20px 20px;
```

Comment:
this does nothing? `padding: 20px;` is identical, u can just remove the second `20px`